5001-6000/LIT5673AkkonuColl.xml

```diff
@@ -209,12 +211,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
 
                 <div type="textpart" xml:id="AkkColl6">
```

Comment:
here you must have the div as it was, with id AkkColl6 and corresp 3592
and then two nested div to the variant miracles with corresp to new IDs